platform_e2e/__init__.py

```diff
@@ -178,15 +178,16 @@ async def check_job_output(
         chunks = []
         while time.monotonic() - started_at < JOB_OUTPUT_TIMEOUT:
             log.info("Monitor %s", job_id)
-            async for chunk in self.client.jobs.monitor(job_id):
-                if not chunk:
-                    break
-                chunks.append(chunk.decode())
-                if re.search(expected, "".join(chunks), re_flags):
-                    return
-                if time.monotonic() - started_at < JOB_OUTPUT_TIMEOUT:
-                    break
-                await asyncio.sleep(JOB_OUTPUT_SLEEP_SECONDS)
+            async with self.client.jobs.monitor(job_id) as it:
+                async for chunk in it:
+                    if not chunk:
+                        break
+                    chunks.append(chunk.decode())
+                    if re.search(expected, "".join(chunks), re_flags):
+                        return
+                    if time.monotonic() - started_at > JOB_OUTPUT_TIMEOUT:
```

Comment:
we previously compared `if time.monotonic() - started_at < JOB_OUTPUT_TIMEOUT:` which led to adding only the first chunk of logs to `chunks` var @zubenkoivan

Reply:
Should we move `chunks = []` inside of external loop? to reset chunks before calling `monitor(job_id)`

Reply:
good catch!